packages/core/content-releases/server/src/services/release.ts

```diff
@@ -28,9 +28,31 @@ const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
       data: releaseWithCreatorFields,
     });
   },
-  findOne(id: GetRelease.Request['params']['id'], query = {}) {
-    return strapi.entityService.findOne(RELEASE_MODEL_UID, id, query);
+
+  async findOne(id: GetRelease.Request['params']['id'], query = {}) {
+    const { createdBy, ...releaseData } = (await strapi.entityService.findOne(
+      RELEASE_MODEL_UID,
+      id,
+      {
+        ...query,
+        populate: ['createdBy'],
+      }
+    )) as unknown as Release & { createdBy: UserInfo };
+
+    const sanitizedCreatedBy = {
+      id: createdBy.id,
+      firstname: createdBy.firstname,
+      lastname: createdBy.lastname,
+      username: createdBy.username,
+      email: createdBy.email,
+    };
```

Comment:
Instead of this you can use the `sanitizeOutput` from the permissionsManager. Also, instead of sanitize at service level I think we can do it, for this case, at a controller level, same for populating the createdBy.

So you can remove the `populate: ['createdBy']` here and then on `.../server/src/controllers/release.ts` findOne function you can do something like this:

```ts
const release = await releaseService.findOne(id, { populate: ['createdBy'] })

const permissionsManager = strapi.admin.services.permissions.createPermissionsManager(...)

const sanitizedRelease = await permissionsManager.sanitizeOutput(release)

ctx.body = { ... }
```